### PR TITLE
Bidder Info Endpoint: Base Adapter Only Filter 

### PR DESCRIPTION
**Pull request by SyntaxNode**
Per: https://github.com/prebid/prebid-server/issues/3079

### CODE REVIEW

File: endpoints/info/bidders_test.go [hunk 1]
```diff
@@ -12,238 +12,470 @@ import (
...
+			expected:            `["c","d"]`,
+		},
+		{
+			name:                "alias-mixed-sorted",
+			givenBidders:        config.BidderInfos{"z": enabledAlias, "a": enabledCore},
+			givenRequestAliases: nil,
+			expected:            `["a","z"]`,
+		},
+		{
+			name:                "defaultequest-one",
```
Comment (hhhjort) [inline endpoints/info/bidders_test.go:232]: Should this and the below be `defaultrequest` rather than `defaultequest`?

Comment (SyntaxNode) [reply endpoints/info/bidders_test.go]: Yes. Yes it should. Nice catch. Fixed.

File: endpoints/info/bidders.go [hunk 2]
```diff
@@ -94,3 +138,32 @@ func prepareBiddersResponseEnabledOnly(bidders config.BidderInfos, aliases map[s
 
 	return json.Marshal(bidderNames)
 }
+
+func prepareBiddersResponseEnabledOnlyBaseOnly(bidders config.BidderInfos) ([]byte, error) {
```
Comment (AlexBVolcy) [inline endpoints/info/bidders.go:142]: Does this function need the word Only in it twice? Would `prepareBiddersResponseEnabledBaseOnly` be better? Or is it better to clearly state that we're looking for only endpoint Enabled and only Base adapters?

Comment (SyntaxNode) [reply endpoints/info/bidders.go:142]: I don't think it matters too much. The names are intentional to match up with the `EnabledOnly` and `BaseOnly` flags.

**Review by AlexBVolcy — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**